package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wstrade-api",
-  "version": "1.9.0",
+  "version": "1.9.1",
```

Comment:
i am open to discussion on this, but i believe the version should be bumped in the minor rather than patch. While we aren't adding a new API, we are extending the functionality of the existing one (auth.`on`), so there is added functionality to the user.